src/main/java/me/eigenraven/lwjgl3ify/textures/StbStitcher.java

```diff
@@ -0,0 +1,79 @@
+package me.eigenraven.lwjgl3ify.textures;
+
+import cpw.mods.fml.common.ProgressManager;
+import me.eigenraven.lwjgl3ify.api.Lwjgl3Aware;
+import net.minecraft.client.renderer.StitcherException;
+import net.minecraft.client.renderer.texture.Stitcher;
+import net.minecraft.util.MathHelper;
+import org.lwjgl.stb.STBRPContext;
+import org.lwjgl.stb.STBRPNode;
+import org.lwjgl.stb.STBRPRect;
+import org.lwjgl.stb.STBRectPack;
+
+@Lwjgl3Aware
+public class StbStitcher {
+    // Returns size
+    public static int packRects(Stitcher.Holder[] holders) {
+        ProgressManager.ProgressBar bar = ProgressManager.push("Stitch setup", holders.length);
+        int holderSize = holders.length;
+
+        // Allocate memory for the rectangles and the context
+        try (
+            STBRPRect.Buffer rectBuf = STBRPRect.calloc(holderSize);
```

Comment:
I don't think you need to zero the buffer or context object.

Reply:
Good catch! I used calloc to make sure that nothing was misallocated, but since this code is initializing everything it needs to I don't think it's needed.